rtgui/locallabtools2.cc

```diff
@@ -8557,18 +8617,13 @@ Locallabcie::Locallabcie():
 
     expprecam->add(*gamcieBox, false);
 
-
+    sigfraBox->pack_start(*modeHBoxbwev);
+    sigfraBox->pack_start(*slopesmoq);
     sigfraBox->pack_start(*sigmoidldacie);
     sigfraBox->pack_start(*sigmoidthcie);
-    sigfraBox->pack_start(*sigmoidsenscie);
```

Comment:
Remove all references to `sigmoidsenscie`.

Reply:
OK but see my post on free software and compatibility